### PR TITLE
Added Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
:-)

### CODE REVIEW

**Comment by juzzlin** [comment]
Thanks! I merged this from CLI, because I had to update also CMakeLists.txt, heimer.pro and application.cpp, too.

File: src/editor/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Translation files in src/editor/translations (without .ts)
-set(TS dustrac-editor_fi dustrac-editor_it dustrac-editor_cs dustrac-editor_fr dustrac-editor_de dustrac-editor_ru)
+set(TS dustrac-editor_fi dustrac-editor_it dustrac-editor_cs dustrac-editor_fr dustrac-editor_de dustrac-editor_ru 
+dustrac-game_nl)
```
Comment (juzzlin) [inline src/editor/CMakeLists.txt:3]: `dustrac-game_nl` added here, but should be `dustrac-editor_nl` I guess..

Comment (Vistaus) [reply src/editor/CMakeLists.txt]: Oops, you're right. Fixed now.

**Comment by Vistaus** [comment]
Ah, I didn't know that. Thanks for the heads-up! I added the requested changes just now :)

**Comment by juzzlin** [comment]
Rebased and merged, thanks!

**Pull request closed without merging**